package.json

```
@@ -54,7 +54,7 @@
     "@vue/compiler-sfc": "^3.0.5",
     "callsite": "^1.0.0",
     "camelcase": "^6.2.0",
-    "cosmiconfig": "^7.0.0",
+    "cosmiconfig": "^8.1.3",
```

```suggestion
    "cosmiconfig": "^8.2.0",
```

Latest is now 8.2.0.

---

Now 8.3.6